integration-test/trino-it/init/hive/init.sh

```diff
@@ -19,6 +19,6 @@
 #
 
 IP=$(hostname -I | awk '{print $1}')
-sed -i "s|<value>hdfs://localhost:9000|<value>hdfs://${IP}:9000|g" /usr/local/hive/conf/hive-site.xml
+sed -i "s|<value>hdfs://__REPLACE__HOST_NAME:9000|<value>hdfs://${IP}:9000|g" ${HIVE_TMP_CONF_DIR}/hive-site.xml
```

Comment:
The sed command seems unnecessary since the `start.sh` already did the replace:
https://github.com/apache/gravitino/blob/1ea4860e8abe047f5469ed39b3f43da2d28cd7e5/dev/docker/hive/start.sh#L35

Reply:
The Trino need to replace the host name to the IP address